packages/webapp/src/actions/drawCard.ts

```diff
@@ -92,8 +92,6 @@ async function drawCardImpl(args: DrawCardArgs): Promise<boolean> {
   const cards = getCards()!
   console.log(`drew card ${cards[selectedCard]}`)
 
-  args.setLoading("Generating draw proof ...")
-
```

Comment:
Something as simple as: 

```
function setLoading(message: string|null|undefined) {
  // Intentionally left blank to ignore any loading message.
  // This function accepts a message parameter but does nothing with it.
}
```
?

Reply:
I think we should handle the case where it is set to null to make the toast go away.

Reply:
The toast disappears in a couple of seconds as is.

Reply:
I'm going to make the call we need to dismiss it — as if it overlaps with the card being highlighted, which is really "eh".

Also, we should "pop up" the hand (as if it was hovered by the cursor, even if it isn't) at least for the time of the card highlight. I would make the card highlight time a tad longer as well (maybe 3s instead of 2.5?)